api/src/handlers/list_validators.rs

```diff
@@ -23,7 +27,7 @@ pub struct Response {
 pub struct QueryParams {
     epochs: Option<usize>,
     query: Option<String>,
-    query_identities: Option<String>,
```

Comment:
let's keep identities and let's have it behave as `query` if present, so we can keep FE compatible until FE switches